pkg/expr/nodes.go

```diff
@@ -111,6 +113,28 @@ func buildCMDNode(rn *rawNode, toggles featuremgmt.FeatureToggles) (*CMDNode, er
 		CMDType: commandType,
 	}
 
+	if true { // toggles.IsEnabledGlobally(featuremgmt.FlagExpressionParser) {
```

Comment:
set to true so it is used in the test pipeline -- not sure how else to support that, and if we want/need more tests here